design/api.go

```diff
@@ -21,7 +21,7 @@ var _ = a.API("alm", func() {
 	})
 	a.Origin("/[.*almighty.io|localhost]/", func() {
```

Comment:
Just curious, what does that thing impact because we've been having `almighty.io` all these weeks and everything looked fine to me :)

If that's the case, we shouldn't be using the name of the redhat offering :)